plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -263,6 +263,7 @@ func ClusterRoles() []rbac.ClusterRole {
 // ClusterRoleBindings return default rolebindings to the default roles
 func ClusterRoleBindings() []rbac.ClusterRoleBinding {
 	rolebindings := []rbac.ClusterRoleBinding{
+		rbac.NewClusterBinding("admin").Groups(user.AdminGroup).BindingOrDie(),
```

Comment:
the "admin" role is intended to be a namespace-limited role. I don't foresee a group granting that role across all namespaces. https://github.com/kubernetes/kubernetes/pull/39537/files#diff-6e3b476b9225d1213dc6ad13e453fc16R225 adds the specific bootstrap admin user to the superuser group, since that is the user that needs to be able to load any other policy and delegate permissions to any other user.